pandas/tests/scalar/timedelta/test_construction.py

```diff
@@ -210,3 +206,28 @@ def test_td_constructor_on_nanoseconds(constructed_td, conversion):
 def test_td_constructor_value_error():
     with pytest.raises(TypeError):
         Timedelta(nanoseconds='abc')
+
+
+class not_raises(object):
+    def __enter__(self):
+        pass
+
+    def __exit__(self, exc_type, exc_val, exc_tb):
+        pass
+
+
+@pytest.mark.parametrize("redundant_unit, expectation", [
+    ("", not_raises()),
+    ("d", pytest.raises(ValueError)),
+    ("us", pytest.raises(ValueError))])
```

Comment:
Is it needed to test that it is raising the value error with all the other combinations? (because this gives a combinatorial explosion of number of test .. ;-))

Reply:
I see, I just wanted to make sure kinda all cases were covered. I believe this does not give a lot of combinations, so I thought that for this case specifically it would be ok.